detections/endpoint/remote_process_instantiation_via_winrm_and_powershell.yml

```
@@ -19,7 +19,7 @@ data_source:
 - CrowdStrike ProcessRollup2
 search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time)
   as lastTime from datamodel=Endpoint.Processes where `process_powershell` (Processes.process="*Invoke-Command*"
-  AND Processes.process="*-ComputerName*") by Processes.action Processes.dest Processes.original_file_name
+  AND (Processes.process="*-ComputerName*" OR Processes.process="*-Cn*")) by Processes.action Processes.dest Processes.original_file_name
```

Adding a space to avoid any rare edge cases.

```suggestion
  AND (Processes.process="*-ComputerName*" OR Processes.process="*-Cn *")) by Processes.action Processes.dest Processes.original_file_name
```